gecko/compose/team.py

```
@@ -132,55 +145,94 @@ async def _worker(idx: int, member: Any, inp: Any):
+                        tg.start_soon(_worker, i, member, inputs[i])
+        except TimeoutError:
+            logger.warning(f"Team {self.name} execution timed out after {timeout}s")
+            raise
+        
+        # 检查超时是否触发
+        if timeout is not None and timeout_cm.cancel_called:  # type: ignore
+            logger.warning(f"Team {self.name} execution timed out")
+            # 返回已完成的结果或超时错误
+            final_results = [r for r in results if r is not None]
+            if not final_results:
+                raise RuntimeError(f"Team {self.name} execution timed out and no member completed")
```

Ruff (0.14.7)

Avoid specifying long messages outside the exception class

(TRY003)